plopfile.mjs

```diff
@@ -0,0 +1,90 @@
+// See https://plopjs.com/
+
+/** @typedef {import('plop').NodePlopAPI} NodePlopAPI */
+/** @typedef {import('plop').ActionConfig} ActionConfig */
+
+/**
+ * @param {string} name
+ * @returns string
+ */
+function templateFile(name) {
+    return `.plop/templates/${name}.hbs`
+}
+
+/** @param {NodePlopAPI} plop */
+function componentGenerator(plop) {
+    plop.setGenerator('component', {
+        description: 'React component with tests, styles, and storybook docs',
+
+        prompts: [
+            {
+                type: 'input',
+                name: 'name',
+                message: 'Component name',
+            },
+            {
+                type: 'input',
+                name: 'location',
+                message: 'Component folder (inside src/)',
```

Comment:
I feel it is worth to leave a note that you probably want to use the default folder here.

Reply:
Maybe we do not need to ask for this info at all. I was always in doubt to include it, and did it mostly as a proof of concept of what's possible.